non-critical-infra/hosts/umbriel/default.nix

```diff
@@ -44,5 +50,39 @@
     linkConfig.RequiredForOnline = "routable";
   };
 
+  # How to generate:
+  #
+  #   $ cd non-critical-infra
+  #   $ SECRET_PATH=secrets/freescout-app-key.umbriel
+  #   $ ssh-keygen -t ed25519 -f "$SECRET_PATH" -P "" -C root@umbriel
+  #   $ sops encrypt --in-place "$SECRET_PATH"
+  #   $ rm "$SECRET_PATH".pub
+  sops.secrets.storagebox-ssh-key = {
+    sopsFile = ../../secrets/storagebox-ssh-key.umbriel;
+    format = "binary";
+    path = "/var/keys/storagebox-ssh-key";
+    mode = "0600";
+    owner = "root";
+    group = "root";
+  };
+
+  sops.secrets.backup-secret = {
+    sopsFile = ../../secrets/backup-secret.umbriel; # <<< TODO: how to generate >>>
```

Comment:
@mweinelt, I read through https://github.com/NixOS/infra/pull/345, but wasn't able to figure out how you generated this for caliban. Can you point me in the right direction?

Reply:
Probably https://github.com/NixOS/infra/pull/345#issuecomment-1914977208 with init instead of list.

Reply:
Ooh, turns out that wasn't necessary because we have [`doInit = true`](https://github.com/jfly/infra/blob/c21bc89ef5fefa200913c8581bef94217ebe78dd/modules/backup.nix#L158-L159)